example/CMakeLists.txt

```diff
@@ -1,35 +1,8 @@
-cmake_minimum_required(VERSION 3.13...3.24)
-project(p4estExamples LANGUAGES C)
-
-include(CheckSymbolExists)
-include(CheckIncludeFile)
-
 option(mpi "use MPI" off)
 
 # --- find external libraries
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../cmake)
```

Comment:
you probably do need this line anymore; same cmake project, CMAKE_MODULE_PATH should be ok, not need to modify it

Reply:
Thanks. This line is removed at the latest state if the branch.